tests/apollo/test_skvbc_commit_path.py

```diff
@@ -85,24 +113,58 @@ async def test_fast_to_slow_path(self, bft_network, tracker):
 
         Finally the decorator verifies the KV execution.
         """
-
+        op_count = int(self.EVALUATION_PERIOD_SEQUENCES * 0.1)
```

Comment:
Why did you choose the 0.1 factor here? Maybe a comment in code would help.

Reply:
Added comment